package.json

```diff
@@ -5,7 +5,8 @@
   "workspaces": [
     "packages/*",
     "l2geth",
-    "integration-tests"
+    "integration-tests",
+    "specs"
```

Comment:
@gakonst is this change sufficient to "add it to the global yarn workspace" as requested?

Reply:
Yes!